pandas/io/common.py

```diff
@@ -805,6 +805,19 @@ def __init__(
         # _PathLike[str]], IO[bytes]]"
         super().__init__(file, mode, **kwargs_zip)  # type: ignore[arg-type]
 
+    # GH39465
```

Comment:
comments would go in a doc-string not above the function

Reply:
I don't know the Python code style super-well, hope what I just pushed fits your needs :)